packages/api/src/derives/nft/tokenInfo.ts

```diff
@@ -51,3 +52,36 @@ export function tokenInfo(instanceId: string, api: ApiInterfaceRx) {
       );
   };
 }
+
+/**
+ * Get info on the current token
+ *
+ * @param owner  The owner address
+ *
+ * @returns [[EnchanceTokenId]]
+ */
+export function allTokenWithOwner(instanceId: string, api: ApiInterfaceRx) {
```

Comment:
Would be nice if this accepted a list of collectionIds to check, otherwise check all
```suggestion
export function tokensOf(instanceId: string, api: ApiInterfaceRx) {
```